### PR TITLE
Add xcuitest doc

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit.md [hunk 1]
```diff
@@ -35,6 +36,7 @@ The toolkit currently supports:
       {"label":"Playwright","value":"playwright"},
       {"label":"TestCafe","value":"testcafe"},
       {"label":"Espresso","value":"espresso"}
```
Comment (FriggaHel) [inline docs/testrunner-toolkit.md:38]:
```suggestion
      {"label":"Espresso","value":"espresso"},
```

File: docs/testrunner-toolkit/configuration/xcuitest.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+## `xcuitest`
+
+__Description__: Details specific to the `xcuitest` configuration.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+xcuit:
```
Comment (FriggaHel) [inline docs/testrunner-toolkit/configuration/xcuitest.md:24]:
```suggestion
xcuitest:
```

**Review by FriggaHel — CHANGES_REQUESTED** [review]

**Review by sweeneyskirt-sl — APPROVED** [review]
This looks fine to me, but I noticed there is no simulators property (like Espresso's emulators). Is it not yet supported for XCUITest?

File: docs/testrunner-toolkit/configuration/xcuitest.md (hunk 2 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/xcuitest.md]: updated

**Comment by tianfeng92** [comment]
@sweeneyskirt-sl `simulators` hasn't been implemented yet. So currently there is no documented field.

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**